docs/upgrade/2.3.2_to_2.4.0.rst

```diff
@@ -79,28 +79,29 @@ until a fix is available (expected on May 31, 2022). If you are using Tor Browse
 for non-SecureDrop browsing, we recommend restarting Tor Browser before and after using it
 for SecureDrop.
 
-Upgrade to Tails 5.0
---------------------
+Upgrade from Tails 4 to Tails 5 series
```

Comment:
nit: can probably just say `Tails 5`

Reply:
shortened in headline, kept verbose in body

Reply:
works for me.